src/test/java/example/Example.java

```diff
@@ -5,6 +5,7 @@
 package example;
```

Comment:
yup. it's in the test package, so it's never consumed. wanted something that was clearly an example and not production code